administrator/language/de-DE/com_joomlaupdate.ini

```
@@ -15,6 +17,8 @@ COM_JOOMLAUPDATE_CONFIG_UPDATESOURCE_DEFAULT="Standard"
 COM_JOOMLAUPDATE_CONFIG_UPDATESOURCE_LABEL="Update-Server"
 COM_JOOMLAUPDATE_CONFIG_UPDATESOURCE_NEXT="Joomla! Next"
 COM_JOOMLAUPDATE_CONFIG_UPDATESOURCE_TESTING="Test"
+COM_JOOMLAUPDATE_CONFIG_VERSIONCHECK_DESC="Zeigt das Kontrollkästchen in der Prüfung vor dem Update an, wenn eine der auf der Website installierten Erweiterungen möglicherweise nicht mit der Version von Joomla kompatibel ist, auf die aktualisiert werden soll. Hinweis: Das Kontrollkästchen wird angezeigt, wenn auf eine neue Joomla-Version (Minor- oder Major-Version) aktualisiert wird."
+COM_JOOMLAUPDATE_CONFIG_VERSIONCHECK_LABEL="Kontrollkästchen für potenziell inkompatible Erweiterungen"
```

```suggestion
COM_JOOMLAUPDATE_CONFIG_VERSIONCHECK_LABEL="Kontrollkästchen für möglicherweise inkompatible Erweiterungen"
```
https://github.com/joomlagerman/joomla/blob/b4e9a051bb42f78757c38cf0eada199fec37daba/administrator/language/de-DE/com_joomlaupdate.ini#L130